docs/learn-crowdloans.md

```diff
@@ -62,3 +60,18 @@ Here is an exmaple of the crowdloans in play during the very first Kusama auctio
 
 Furthermore, check out this video on [How to Participate in Crowdloans](https://www.youtube.com/watch?v=YrTxDufrcQM)
 for steps on how to access available crowdloans on PolkadotJS apps.  
+
+## Starting a crowdloan campaign
```

Comment:
This section would be better suited with the above content that discusses the creation of a crowdloan.

With this change, it would seem to me that the best approach would be to have two sections, one for creators of the crowdloan and one for participants. For instance:

`## Starting a crowdloan campaign`

---

`## Supporting a crowdloan campaign`

...
what do you think?

Reply:
I agree, I tried to separate the document into two parts as you suggested, I hope it works.